docs/release-notes/cloud/cloud-2023-10.md

```diff
@@ -0,0 +1,81 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Cloud October 2023.
+included_jira_versions: ['2023.10']
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/7.10.32
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/7.10.80
+---
+
+# Cloud October 2023
+
+These release notes are for the Codacy Cloud updates during October 2023.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Product enhancements
+
+-   You can now get two additional coverage status reports on GitHub using the new faster Coverage engine, showing whether your diff coverage and coverage variation are up to standards or not as configured on the quality gate rules for your repository. See [how to activate this setting](../../repositories-configure/integrations/github-integration.md#new-coverage-engine) for a repository. (ALA-593)
+
+    ![New Coverage status report](../images/ala-593.png)
+
+-   Codacy will drop support for branches named HEAD or matching the pattern refs/heads/*. These names are reserved for git and shouldn't be used as branch names. (IO-830)
+
+## Bug fixes
+
+-   Fixed an issue that caused code duplication warnings to link to the original code block instead of the duplicated one. (TS-545)
+-   TSQLLint can now also detect and analyze files with an `sql` extension. (TS-543)
+-   Analysis retries now follow the same permissions configured in Analysis Configuration (configurable in Organization settings / Member privileges / Analysis configuration) allowing a members to actually ignore issues since they can now reanalyse commits and pull requests (PLUTO-587)
```

Comment:
@claudiacarpinteiro Can you review this one? 🙏

Reply:
@nicklem I already clarified with @lolgab, this one is a **product enhancement** (although it started as a bug fix). 

Here's my review:
```suggestion
-   The permission level set for the **Analysis configuration** on the organization **Settings > Member privileges** now applies also to the reanalysis of a pull request or the last commit in a branch. (PLUTO-587)
```